code/modules/atmospherics/machinery/components/binary_devices/thermomachine.dm

```diff
@@ -300,6 +352,23 @@
 	update_appearance()
 	return TRUE
 
+/obj/machinery/atmospherics/components/binary/thermomachine/proc/check_explosion(temperature)
+	if(temperature < FUSION_MAXIMUM_TEMPERATURE + 2000)
+		return FALSE
+	if(prob(log(15, temperature) * 10)) //68% at 1e8, 100% at 1e12
```

Comment:
can you make the hardcap 1e8? or is that too low. it's quite nice to have a consistent sanity limit about there

Reply:
sure, is not too low, i'll place the safeties cap around 1e5/1e6